go.mod

```diff
@@ -107,16 +108,17 @@ require (
 	go.etcd.io/bbolt v1.3.5
 	go.uber.org/goleak v1.1.10
 	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	golang.org/x/oauth2 v0.0.0-20210402161424-2e8d93401602
 	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
-	google.golang.org/api v0.25.0
-	google.golang.org/genproto v0.0.0-20201022181438-0ff5f38871d5
-	google.golang.org/grpc v1.33.1
+	google.golang.org/api v0.44.0
+	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c
+	google.golang.org/grpc v1.38.0
 	google.golang.org/protobuf v1.26.0
 	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
 	k8s.io/api v0.19.4
 	k8s.io/apimachinery v0.19.4
 	k8s.io/client-go v0.19.4
+	sigs.k8s.io/structured-merge-diff/v4 v4.0.2 // indirect
```

Comment:
I think this is unrelated? I didn't end up with this change when I bumped buildpack to `v0.20.0` 🤔 

```
brian@localghost:waypoint λ go get github.com/buildpacks/pack@v0.20.0                                                                                                                                ±[main]
go get: upgraded github.com/buildpacks/pack v0.18.1 => v0.20.0
go get: upgraded github.com/google/go-cmp v0.5.5 => v0.5.6
go get: upgraded golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d => v0.0.0-20210402161424-2e8d93401602
go get: upgraded google.golang.org/api v0.25.0 => v0.44.0
go get: upgraded google.golang.org/genproto v0.0.0-20201022181438-0ff5f38871d5 => v0.0.0-20210602131652-f16073e35f0c
go get: upgraded google.golang.org/grpc v1.33.1 => v1.38.0
```

Reply:
Forgive me, but how exactly are you bumping? 😅 All I did was:

```shell
go get github.com/buildpacks/pack@v0.20.0
```

Reply:
I ran the same command against `main`:

> brian@localghost:waypoint λ go get github.com/buildpacks/pack@v0.20.0  

I'm not sure what's up, you might of had previous changes staged? No worries, I can always open a PR to fix this too if you can't get it to play nice! Let me know ❤️

Reply:
It should be resolved in the latest commit. Please give it another look.